frontend/lib/rental-history.tsx

```diff
@@ -29,13 +30,25 @@ function RentalHistoryWelcome(): JSX.Element {
   }
 
 function RentalHistoryForm(): JSX.Element {
+  const appContext = useContext(AppContext);
+  const userData = appContext.session;
+
+  const UserRhFormInput: RhFormInput = (userData && userData.userId ?
```

Comment:
oh you might want to consider lowercasing the first character of this... I am actually not entirely sure why I capitalized the first letter of `BlankRhFormInput` in the querybuilder code...  arg, inconsitencies!  In any case it's up to you.  I think that true `CamelCase`-style names (with the first letter capitalized) should only be used for top-level objects that are either classes or "singleton" objects (I sort of think of them as classes where all members are static).